### PR TITLE
Fix links that have `support` in the path

### PR DESCRIPTION
**Pull request by bluwave**
refs: MBL-14220
affects: Student
release note: none

### CODE REVIEW

File: Student/Student/Routes.swift [hunk 1]
```diff
@@ -352,9 +352,14 @@ let routeMap: KeyValuePairs<String, RouteHandler.ViewFactory?> = [
         return ProfileSettingsViewController.create()
     },
 
-    "/support/:type": { _, params in
-        let type = params["type"].flatMap { ErrorReportType(rawValue: $0) } ?? .problem
-        return ErrorReportViewController.create(type: type)
+    "/support/:type": { url, params in
+        if url.host?.isEmpty == false {
+            open(url: url)
```
Comment (bluwave) [inline Student/Student/Routes.swift:357]: is there a better to forward this on to safari?

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/15db8wj) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F15db8wj)
teacher: [Link](https://rebrand.ly/d9mp9au) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fd9mp9au)
parent: [Link](https://rebrand.ly/vo7xc16) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fvo7xc16)

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Student

[MBL-14220](https://instructure.atlassian.net/browse/MBL-14220)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.57% | 93.59% | -0.02%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/Router/Router.swift [hunk 2]
```diff
@@ -181,6 +181,12 @@ public class Router: RouterProtocol {
         DeveloperMenuViewController.recordRouteInHistory(url.url?.absoluteString)
         #endif
         Analytics.shared.logEvent("route", parameters: ["url": String(describing: url)])
+
+        if url.host?.isEmpty == false && !urlMatchesSessionHost(url) {
```
Comment (vanwagonet) [inline Core/Core/Router/Router.swift:185]: awesome. This way we never route to an external url that happens to match one of our paths.

**Pull request merged**